packages/forms/src/fields/ObjectField.js

```diff
@@ -103,6 +103,9 @@ class ObjectField extends Component {
 		const { widget, ...options } = getUiOptions(uiSchema);
 		// widget
 		if (typeof widget === 'string') {
+			if (widget === 'hidden') {
+				return (<input type='hidden'/>);
```

Comment:
I'm not sure that implementation will actually work.
The main issue is that an object on the JsonSchema doesn't represent an actual input. I'm not sure if it's a good idea to replace a collection of inputs by by single hidden one.